eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -1603,3 +1628,49 @@ submitYesVoteCCs_ ::
   ImpTestM era ()
 submitYesVoteCCs_ committeeMembers govId =
   mapM_ (\c -> submitYesVote_ (CommitteeVoter c) govId) committeeMembers
+
+submitUpdateCommittee ::
+  ConwayEraImp era =>
+  -- | Set the parent. When Nothign is supplied latest parent will be used.
+  Maybe (StrictMaybe (GovPurposeId 'CommitteePurpose era)) ->
+  [(Credential 'ColdCommitteeRole (EraCrypto era), EpochInterval)] ->
+  UnitInterval ->
+  ImpTestM era (GovActionId (EraCrypto era))
+submitUpdateCommittee mParent ccs threshold = do
+  nes <- getsNES id
+  let
+    curEpochNo = nes ^. nesELL
+    rootCommittee =
+      nes ^. nesEsL . esLStateL . lsUTxOStateL . utxosGovStateL . cgsProposalsL . pRootsL . grCommitteeL
+    parent = fromMaybe (prRoot rootCommittee) mParent
+    newCommitteMembers =
+      Map.fromList [(cc, addEpochInterval curEpochNo lifetime) | (cc, lifetime) <- ccs]
+  submitGovAction $ UpdateCommittee parent mempty newCommitteMembers threshold
+
+expectCommitteeMemberPresence ::
+  (HasCallStack, ConwayEraGov era) => Credential 'ColdCommitteeRole (EraCrypto era) -> ImpTestM era ()
+expectCommitteeMemberPresence cc = do
+  SJust committee <-
+    getsNES $ nesEsL . esLStateL . lsUTxOStateL . utxosGovStateL . committeeGovStateL
+  assertBool ("Expected Committee Member: " ++ show cc ++ " to be present in the committee") $
+    Map.member cc (committee ^. committeeMembersL)
```

Comment:
I actually thought about it. There are two downsides to this:

1. When no committee is present it will result in a different failure (pattern match on `SNothing`), which will make it clear that we are in the state of no confidence
2. Much less important for testing, but we would do a redundant conversion to a Set.

Reply:
In other words there is a difference between an empty comittee and no committee and `getCommitteeMembers` erases that difference